centos2ol.sh

```diff
@@ -303,11 +303,25 @@ case "$os_version" in
         fi
         ;;
     8*)
-        # Workaround for qemu-guest-agent packages installed from virt modules
-        if rpm -q qemu-guest-agent; then
-            dnf module reset -y virt
-            dnf module enable -y virt
-            dnf install -y --allowerasing qemu-guest-agent
+        # There are a few dnf modules that are named after the distribution
+        #  for each steam named 'rhel' or 'rhel8' perform a module reset and install
+        mapfile -t modules_enabled < <(dnf module list --enabled | grep rhel | cut -f1 -d\  )
+        if [[ "${modules_enabled[*]}" ]]; then
+            for module in "${modules_enabled[@]}"; do
+                dnf module reset -y "${module}"
+                case ${module} in
+                container-tools|go-toolset|jmc|llvm-toolset|rust-toolset)
+                    dnf module install -y "${module}":ol8
+                    ;;
+                virt)
+                    dnf module install -y "${module}":ol
+                    ;;
+                *)
+                    echo "Unsure how to transform module ${module}"
+                    ;;
+                esac
+            done
+            dnf update -y --disablerepo "*" --enablerepo "ol8_appstream"
```

Comment:
Extremely nitpick, but can you reorder the parameters so that the `update` directive is last, i.e.

`dnf --disablerepo="*" --enablerepo="ol8_appstream" --assumeyes update`

The switch from `-y` to `--assumeyes` is for consistency with the other two other long-form parameters.